libopae++/include/opaec++/pvalue.h

```diff
@@ -126,16 +141,20 @@ struct pvalue {
   friend std::ostream &operator<<(std::ostream &ostr, const pvalue<T> &p) {
     T value;
     fpga_properties props = *p.props_;
-    if (p.get_(props, &value) == FPGA_OK) {
+    fpga_result res;
+    if ((res = p.get_(props, &value)) == FPGA_OK) {
       ostr << +(value);
     } else {
-      // TODO: Log or throw
+      p.log_.error() << "property getter failed with (" << res
+                     << ") " << fpgaErrStr(res);
+      throw except(res, OPAECXX_HERE);
```

Comment:
What if the `res` variable is `FPGA_NOT_FOUND`?

Reply:
Yes, thanks.